### DISCUSSION TITLE
Seems there might be a bug here nest auth is saying that the strategy in authprovider shouldn't be sting but the docs show it as a string

### DISCUSSION THREAD

**Discussion by wakywayne** [nextauthjs/next-auth#6281] [Help]
### Question 💬

example code:

```
    session: {
        strategy: "jwt",
    },
```  

typescript error:

```
Argument of type '{ providers: (EmailConfig | OAuthConfig<GoogleProfile> | OAuthConfig<FacebookProfile>)[]; adapter: Adapter<boolean>; ... 4 more ...; callbacks: { ...; }; }' is not assignable to parameter of type 'NextAuthOptions'.


  The types of 'session.strategy' are incompatible between these types.
    Type 'string' is not assignable to type 'SessionStrategy | undefined'.
```

### How to reproduce ☕️

In nextAuth set session strategy to :
docs can be viewed here [JWT auth stradegy](https://next-auth.js.org/configuration/options#:~:text=Default%20values%20for%20this%20option%20are%20shown%20below%3A)

```
    session: {
        strategy: "jwt",
    },
```

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by balazsorban44** [comment]
I am unable to reproduce this, please add a reproduction.

**Comment by wakywayne** [comment]
> I am unable to reproduce this, please add a reproduction.

my repository has been made public here : https://github.com/wakywayne/wrestling-event-coordinator

Or Would you like me to copy paste my whole [...nextauth].js file?

**Comment by balazsorban44** [comment]
I still can't reproduce this. Could you create a *minimal* reproduction only focusing on the type issue, removing absolutely everything else?

**Comment by wakywayne** [comment]
I just need to import the auth options type

**Comment by ChristopherHButler** [comment]
If you type your AuthOptions it should fix it. ref: https://stackoverflow.com/questions/74244256/type-string-is-not-assignable-to-type-sessionstrategy-undefined

```typescript
import type { NextAuthOptions } from "next-auth";

export const authOptions: NextAuthOptions = {
...
}
```
